modules/docs/mdoc/faq.md

```diff
@@ -96,3 +96,14 @@ https://keys.openpgp.org/search?q=me@scala-steward.org.
 ## How can I change log levels?
 
 If you want to switch from the standard INFO log level you can call scala steward with `-DLOG_LEVEL=DEBUG` for example: `scala-steward -DLOG_LEVEL=DEBUG  --workspace  "$STEWARD_DIR/workspace" ...` 
+
+## Why doesn't self-hosted Scala Steward close obsolete PRs?
+
+When a new version of a dependency is found and the PR for older version still exists, Scala Steward 
+will be able to close the old PR.
+
+If you don't observe such behaviour and new version PRs pile up alongside old ones,
+make sure that the workspace folder is persisted between different runs. 
```

Comment:
You  could link `workspace` to the section in `running.md`
https://github.com/keynmol/scala-steward/blob/mention-workspace/modules/docs/mdoc/running.md#workspace

Reply:
ah, good idea!

Reply:
One question - I see both modules/docs/mdoc/faq.md and docs/faq.md - which one should I update?

Reply:
Update `modules/docs/mdoc/faq.md` and then run `sbt docs/mdoc`. `docs/faq.md` is generated from the former, see https://github.com/scala-steward-org/scala-steward/blob/68b7f999125305cfd0a058f89da35999248f406c/build.sbt#L223-L235

Reply:
Let me know if https://github.com/scala-steward-org/scala-steward/pull/2886/commits/0fa21c7f33d2b91132b88e31d568e48932c3feff is good enough, happy to iterate :)